appSyncClient.go

```diff
@@ -159,10 +159,10 @@ func (client *appSyncClient) CreateOrUpdateResolvers(apiID string, resolversFile
 			})
 
 			if err != nil {
-				logger.Println(fmt.Sprintf("Function %s failed to create: %s", *function.Name, err))
+				logger.Println(fmt.Sprintf("Function %s failed to create: %s", function.Name, err))
```

Comment:
`functionName` is Already an existing variable that contains the function name.

Reply:
In the end I removed the functionName variable. There are plenty of references to the function variable... so felt better to just do that instead of creating separate functionName variable